pkg/splunk/util/secrets_test.go

```diff
@@ -428,7 +428,7 @@ func TestGetVersionedSecretVersion(t *testing.T) {
 
 	// Negative testing with non-integer version
 	for testVersion := 0; testVersion < 10; testVersion++ {
-		testSecretName = splcommon.GetVersionedSecretName(versionedSecretIdentifier, string('A'-1+testVersion))
+		testSecretName = splcommon.GetVersionedSecretName(versionedSecretIdentifier, string(rune('A'-1+testVersion)))
```

Comment:
Had to use `rune` to fix a compilation error if using go version >=1.15. This was the error when we ran `make test` -
```pkg/splunk/util/secrets_test.go:431:80: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)```